idl/chromadb/proto/compactor.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
Maybe as future work it would be useful to add an endpoint to see the status for the *currently* compacting and *last N compactions*

Reply:
I.e

`/compaction/status`

returns

```
id: <QUEUED | RUNNING | FAILED | SUCCEEDED> + Time